README.md

```diff
@@ -1,2 +1,45 @@
 # DocumentClass
+
+## Description 
+
 Repository describing unified classes representing the structural elements of a text document
+
+## Install
+
+
+## How use
+
+```python
+from Paragraph import Paragraph
+from helpers.EnumFill import EnumFill
+
+
+para = Paragraph(text=" 123 ", indent=" 1", lineSpacing=1, fontName=" 123", textSize='12',
```

Comment:
I think we need to remove unnecessary spaces from the initialization of the object